generator-input/tweaks/Directory.Packages.props

```diff
@@ -0,0 +1,29 @@
+<Project>
```

Comment:
Yikes, what makes certain these are the same?

Can't you have the root Directory.Packages.props and global.json point to these somehow?